docs/template-bugReport.md

```diff
@@ -0,0 +1,29 @@
+**Bug Report**
+
+**Issue Title:** Bug Report: [Brief description of bug]
+
+**Environment (dev/staging/live, version)**
+
+> Tell us what environment you encountered the bug on,
+>
+> e.g. live server at `V5.93`
+>
+> e.g. `master` branch at commit 1234567
+
+**Steps to Reproduce**
+
+> Help us to reproduce the bug on our end so we can fix it, e.g.
+> 1. Login as an student on the mobile website.
```

Comment:
I think we need a line break before this line, otherwise GitHub won't correctly display this as a list.